free-programming-books-tr.md

```diff
@@ -64,6 +65,9 @@
 
 * [Go El Kitabı](https://github.com/umutphp/the-little-go-book) - Karl Seguin, Umut Işık tarafından çevirildi
 
+### Güvenlik ve Gizlilik
+
+* [Özgür Yazılım Derneği Güvenlik Rehberi](https://guvenlik.oyd.org.tr/)
 
 ### Html
```

Comment:
Hello, thank you for this resource, however there are a few errors to fix before this can be merged.

- There needs to be an extra blank line between the last resource(Go El Kitabi) in the "Go" section and the "Güvenlik ve Gizlilik" section you created
- There needs to be the trailing slash removed from the end of the resource you added. I also went ahead and included the contributing authors as well!
- Similarly, there needs to be an extra blank line between Özgür Yazılım Derneği Güvenlik Rehberi and "Html"

Fixing these would let your PR be merged.

```suggestion
### Go

* [Go El Kitabı](https://github.com/umutphp/the-little-go-book) - Karl Seguin, Umut Işık tarafından çevirildi


### Güvenlik ve Gizlilik

* [Özgür Yazılım Derneği Güvenlik Rehberi](https://guvenlik.oyd.org.tr) - Filiz Akin et al.


### Html
```